File: app/src/main/java/com/dpis/module/settings/ExperimentalSettingsActivity.java
package com.dpis.module.settings;

import com.dpis.module.LocalizedActivity;
import com.dpis.module.ui.compose.SupportActivityContent;

import android.os.Bundle;

public final class ExperimentalSettingsActivity extends LocalizedActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        SupportActivityContent.installExperimentalSettings(this);
    }
}

File: app/src/main/java/com/dpis/module/settings/ExperimentalSettingsActivity.kt
package com.dpis.module.settings

import android.os.Bundle
import com.dpis.module.settings.presentation.installExperimentalSettings

class ExperimentalSettingsActivity : LocalizedActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        installExperimentalSettings()
    }
}
